app/lib/hpitt/code_sets/ethnicities.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module HPITT
+  module CodeSets
+    module Ethnicities
+      MAPPING = {
+        "Asian or Asian British\n(includes any Asian background, for example, Bangladeshi, Chinese, Indian, Pakistani)" => Diversities::ETHNIC_GROUP_ENUMS[:asian],
```

Comment:
Ed and I were discussing having some sensible rules for this sort of stuff in the future. I wonder if it makes to just strip anything that isn't a letter, downcase and compare? That way we'd always do a lookup on the word/combo string itself eg:

`White\n(includes any White background)` => `whiteincludesanywhitebackground` then if there's another key with some slight variation it'll be normalised to match (this doesn't cover mis-spellings of course but might improve the lookup)

Reply:
Yep, sounds sensible, let me try adding that in.

Reply:
Updated via https://github.com/DFE-Digital/register-trainee-teachers/pull/1412/commits/904035ecabb95cea8516875fd3a6f72ff6d56b1a
I'm storing the codeset in a normalised form as well. but happy to change it to call it on the fly if we prefer storing the longer form for readability.